### PR TITLE
DAOS-11149 pool: bump GC ULT stack size

### PR DESCRIPTION
**Pull request by bfaccini**
Looks like both GC/Flush ULT stacks can overflow upon certain
situations, and this may be due to some PMDK/SPDK/DPDK recent
changes ...
So bump their stack size from default/16K to DSS_DEEP_STACK_SZ.

Signed-off-by: Bruno Faccini <bruno.faccini@intel.com>

### CODE REVIEW

**Review by jolivier23 — APPROVED** [review]

**Comment by bfaccini** [comment]
@NiuYawei and @gnailzenh would be cool if you could review when you have some time ;-)

File: src/pool/srv_target.c [hunk 1]
```diff
@@ -198,7 +198,7 @@ start_gc_ult(struct ds_pool_child *child)
 	sched_req_attr_init(&attr, SCHED_REQ_GC, &child->spc_uuid);
 	attr.sra_flags = SCHED_REQ_FL_NO_DELAY;
 
-	child->spc_gc_req = sched_create_ult(&attr, gc_ult, child, 0);
+	child->spc_gc_req = sched_create_ult(&attr, gc_ult, child, DSS_DEEP_STACK_SZ);
```
Comment (NiuYawei) [inline src/pool/srv_target.c:201]: It looks to me unlikely a stack overrun issue, I'm wondering if DAOS-11149 is dup of DAOS-11092. Have @mjean308 verified that PR-9667 (from DAOS-11092) doesn't fix DAOS-11149?

**Comment by brianjmurrell** [comment]
You might want to consider merging master to get more success at passing tests;

_This branch is 2 commits ahead, 54 commits behind master._

File: src/pool/srv_target.c (hunk 1 above)
Comment (bfaccini) [reply src/pool/srv_target.c:201]: Well, I am already waiting for @mjean308 to verify if this PR-9924 doesn't fix DAOS-11149 ??!! ;-)

**Comment by bfaccini** [comment]
Abandoning this debug PR, now that problem has been identified and it is suspected to be fixed by [PR-9667](https://github.com/daos-stack/daos/pull/9667) which has landed.

**Comment by bfaccini** [comment]
Looks like these changes must be pushed finally, as found in [DAOS-11832](https://github.com/daos-stack/daos/pull/9924) associated coredumps.

File: src/pool/srv_target.c (hunk 1 above)
Comment (NiuYawei) [reply src/pool/srv_target.c:201]: I still don't quite see why the unmap call could cause stack overrun (see my comment in ticket), but let's see if this PR could solve the issue.

**Review by NiuYawei — APPROVED** [review]

**Comment by bfaccini** [comment]
The only 2 tests failures for the last CI session : 

```
Existing failures - 2
Test Hardware / Functional Hardware Medium / 3-./osa/online_extend.py:OSAOnlineExtend.test_osa_online_extend_oclass;run-aggregation-checksum-container-daos_racer-extra_servers-hosts-ior-client_processes-iorflags-job_manager-loop_test-mdtest-wr_size-32K-pool-rebuild-server_config-servers-0-1-setup-test_obj_class-test_ranks-9fd5 – FTEST_osa.OSAOnlineExtend

Test Hardware / Functional Hardware Medium / 4-./osa/online_extend.py:OSAOnlineExtend.test_osa_online_extend_mdtest;run-aggregation-checksum-container-daos_racer-extra_servers-hosts-ior-client_processes-iorflags-job_manager-loop_test-mdtest-wr_size-32K-pool-rebuild-server_config-servers-0-1-setup-test_obj_class-test_ranks-9fd5 – FTEST_osa.OSAOnlineExtend

```
appear definitely related to [DAOS-11936](https://daosio.atlassian.net/browse/DAOS-11936) and its associated fix/[PR-10604](https://github.com/daos-stack/daos/pull/10604) is currently under review ...

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**